README.md

```diff
@@ -1,5 +1,5 @@
 # Simplenote for Android
-[![Version](https://img.shields.io/badge/version-2.19-blue.svg)](https://github.com/Automattic/simplenote-android/releases/tag/2.19) [![CircleCI](https://img.shields.io/circleci/build/gh/Automattic/simplenote-android.svg?label=circleci)](https://circleci.com/gh/Automattic/simplenote-android) [![Travis CI](https://img.shields.io/travis/Automattic/simplenote-android/develop.svg?label=travisci)](https://travis-ci.org/Automattic/simplenote-android)
+[![Version](https://img.shields.io/badge/version-2.19-blue.svg)](https://github.com/Automattic/simplenote-android/releases/tag/2.19) [![CircleCI](https://img.shields.io/circleci/build/gh/Automattic/simplenote-android.svg?label=circleci)](https://circleci.com/gh/Automattic/simplenote-android)
```

Comment:
For what is worth, I'd say we should remove the version and the CircleCI shields, too. Those shields provide information that is already available on GitHub:

<img width="1252" alt="Screen Shot 2021-08-23 at 1 06 44 pm" src="https://user-images.githubusercontent.com/1218433/130384619-83d2d9a7-1d2a-48b9-8914-1d6de59a3a5d.png">

Reply:
I agree, let's remove them all.

This information (CI status and latest version) was not as visible a while ago with previous GitHub UI, but now that it is, it's not really relevant anymore.